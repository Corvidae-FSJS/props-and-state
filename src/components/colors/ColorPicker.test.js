import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from '../colors/ColorPicker';

describe('ColorPicker component', () => {
  it('Renders ColorPicker', () => {
    const wrapper = shallow(<ColorPicker colors={[
      'white', 
      'red', 
      'orange', 
      'yellow', 
      'green', 
      'blue', 
      'indigo', 
      'violet', 
      'black'
    ]} selectColor={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
}); 