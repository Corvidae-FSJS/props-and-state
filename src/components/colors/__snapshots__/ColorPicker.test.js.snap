// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker component Renders ColorPicker 1`] = `
<Fragment>
  <button
    key="white"
    onClick={[Function]}
  >
    white
  </button>
  <button
    key="red"
    onClick={[Function]}
  >
    red
  </button>
  <button
    key="orange"
    onClick={[Function]}
  >
    orange
  </button>
  <button
    key="yellow"
    onClick={[Function]}
  >
    yellow
  </button>
  <button
    key="green"
    onClick={[Function]}
  >
    green
  </button>
  <button
    key="blue"
    onClick={[Function]}
  >
    blue
  </button>
  <button
    key="indigo"
    onClick={[Function]}
  >
    indigo
  </button>
  <button
    key="violet"
    onClick={[Function]}
  >
    violet
  </button>
  <button
    key="black"
    onClick={[Function]}
  >
    black
  </button>
</Fragment>
`;
